src/Command/FetchConferencesCommand.php

```diff
@@ -58,171 +41,8 @@ protected function configure(): void
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $io = new SymfonyStyle($input, $output);
+        $this->conferencesHarvester->harvest($io);
```

Comment:
This does not make sens to pass $io to a service. What will happen if you use this service from a web SAPI?